airflow/README.md

```diff
@@ -16,22 +17,46 @@ Download METS file for binding 379973 and all ALTO files listed in the downloade
 
     ```echo -e "AIRFLOW_UID=$(id -u)" > .env```
 
-3. Initialize Airflow in docker:
+3. In order to run the Puhti dag, also add the absolute path to your `.ssh` folder to the `.env` file in the following manner:
+
+    ```SSH_PATH=/your/path/.ssh```
+
+4. Initialize Airflow in docker:
 
     ```docker compose up airflow-init```
 
-4. Build image and run container:
+5. Build image and run container:
 
     ```docker compose up --build```
 
-5. Wait for the container to launch, and open Airflow in `http://0.0.0.0:8080/`
+6. Wait for the container to launch, and open Airflow in `http://0.0.0.0:8080/`
+
+7. Login with the created default username `airflow` and password `airflow`
+
+8. In order for the `download_altos_for_binding_to_puhti` DAG to function, you need to configure two connections in Admin -> Connections:
+
+    1. SSH Connection to Puhti:
+
+        - Select "Add a new record" from the **+** sign
+        - Set Connection Id to `puhti_conn`
+        - Set Connection Type to `SSH`
+        - Set Host to `puhti.csc.fi`
+        - Set your Puhti username as Username
+        - In the Extra-field, add the absolute path to your private SSH key (of which pair is in Puhti) in the following manner:   
+            ```{"key_file": "/your/path/.ssh/id_rsa"}```
 
-6. Login with the created default username `airflow` and password `airflow`
+    2. HTTP Connection to NLF:
```

Comment:
As we don't still have the proper credentials for Puhti login, the sub-step 1 makes sense, but why we do this one manually? Based on `git grep nlf_http`, it seems that we check that user-defined connection, but then when it's time to do the actual API requests, we do those against what is defined in  `DC_IDENTIFIER` and `BASE_PATH`.

Reply:
This HTTP connection to NLF is only needed for the `HTTPSensor` operator that checks the API availability before making tons of requests. You're right, it's not needed for the actual downloading. 

If the sensor seems unnecessary, it can of course be taken out, but it is a pretty nice way to cancel the pipeline immediately if the API is down or something.

Reply:
Ah, I see now, I didn't even think of automating the creation of the connection. Doesn't look too complicated, I'll look into it now.

Reply:
Done!